src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.cs

```diff
@@ -198,12 +167,12 @@ public string Filter
         {
             get
             {
```

Comment:
I suggested that the getter throw off the is more than one in the list. Returning the first of many adds no value. Did API review mtg discuss this?

Reply:
yeah i brought that up during the review mtg , Immo read ur comment but i forget the reason why we declined that

Reply:
We didn't throw because we never threw before. It was a matter of consistency- you can't get this property to throw.